static/app/components/events/viewHierarchy/utils.tsx

```diff
@@ -118,3 +118,17 @@ export function getDeepestNodeAtPoint(
 
   return clickedNode;
 }
+
+// Adapted from static/app/profiling/gl/utils.ts
+export function getCenterScaleMatrixFromConfigPosition(
+  scale: number,
+  center: vec2
+): mat3 {
+  const invertedConfigCenter = vec2.fromValues(-center[0], -center[1]);
+
+  const centerScaleMatrix = mat3.create();
+  mat3.fromTranslation(centerScaleMatrix, center);
+  mat3.scale(centerScaleMatrix, centerScaleMatrix, vec2.fromValues(scale, scale));
+  mat3.translate(centerScaleMatrix, centerScaleMatrix, invertedConfigCenter);
+  return centerScaleMatrix;
+}
```

Comment:
nit: not a blocker, but I feel like we could have updated the original `getCenterScaleMatrixFromConfigPosition` from profile to accept two scalars instead of just one so we could avoid mostly duplicating this function? Unless profiling has a reason not to do that. It's confusing to have two functions of the same name with slightly different functionality

Reply:
Great suggestion! At the time I wanted to avoid changing the util in profiling, but now I've noticed that there aren't that many call sites so it was an easy change to get this